class/defaults.yml

```diff
@@ -1,9 +1,10 @@
 parameters:
   nfs_subdir_external_provisioner:
     namespace: syn-nfs-subdir-external-provisioner
+    multi_instance: true
     release_name: platform
```

Comment:
@simu Is there a possibility to set the release_name to the alias?

Reply:
Yes, you can use

```
    release_name: ${_instance}
````

Reply:
Added `release_name: ${_instance}`